bot/cogs/models/waifulistapi.py

```diff
@@ -249,15 +249,18 @@ async def get(self, endpoint: str) -> Response:
         )
         return r
 
-    async def post(self, endpoint: str, json: dict) -> Response:
+    async def post(self, endpoint: str, json: dict, attempts=0) -> Response:
         """Sends a POST request to Mywaifulist API endpoint."""
         r = await self.client.post(
             API_URL + endpoint, json=json,
             headers=await self.forged_headers, cookies=await self.forged_cookies
         )
+        r.status_code = 419
         if r.status_code == 419:
+            if attempts > 2:
+                raise Exception('Re-authorization attempts failed too many times')
```

Comment:
It should not raise such a generic Exception, you could use APIException as defined in utils/exceptions.py

Reply:
i thought those were reserved for theotown api errors